.trivyignore

```diff
@@ -0,0 +1,3 @@
+# This concerns a dev dependency that is used by @angular-devkit/build-angular.
```

Comment:
`@angular-devkit/build-angular` itself is the dev dependency.

It is a tertiary dependency, that we should also name, meaning we should talk about minimist 2.0.0 that was causing this problem.